rust/syntect-plugin/Cargo.toml

```diff
@@ -19,7 +20,7 @@ toml = "0.4"
 git = "https://github.com/trishume/syntect.git"
 branch = "master"
 default-features = false
-features = ["parsing", "assets","dump-load-rs", "metadata"]
+features = ["parsing", "yaml-load", "assets","dump-load-rs","dump-create", "metadata"]
```

Comment:
I'd double check that yaml-load is necessary, after you've made some other changes

Reply:
`SyntaxSet::load_from_folder` is dependent on it actually.